src/sections/contact-section.html

```diff
@@ -0,0 +1,38 @@
+<div class="pt-[6%] container grid text-2xl grid-rows-2 md:grid-cols-2 gap-16 md:gap-20">
+    <div class=" flex flex-col gap-8 md:gap-20">
+        <div>
+            <h4 class="md:text-xl xl:text-2xl font-bold">IMPRINT</h4>
+            <h2 class="md:text-3xl xl:text-5xl font-bold">Agentur Baumeister</h2>
+            <p class="md:text-3xl xl:text-5xl">Katrein Baumeister</p>
+            <p class="md:text-3xl xl:text-5xl">Böckhstraße 13</p>
+            <p class="md:text-3xl xl:text-5xl">10967 Berlin</p>
+        </div>
+
+        <div class="md:text-3xl xl:text-5xl">
+            <p>+49 (0)30 - 490 827 87</p>
+            <p>hallo@agentur-baumeister.com</p>
+        </div>
+
+        <div class="md:text-3xl xl:text-5xl">
+            <h2 class=" font-bold">UST-IDNR.</h2>
+            <p>DE 274 486 923</p>
+        </div>
+
+        <div>
+            <h2 class="md:text-3xl xl:text-5xl font-bold">REGISTERED OFFICE</h2>
+            <p class="md:text-3xl xl:text-5xl">Berlin, Deutschland</p>
+        </div>
+    </div>
+
+    <div class="flex flex-col md:justify-end md:items-end md:text-end md:text-xl xl:text-2xl md:gap-8">
+        <h2 class="font-bold">DISCLAIMER</h2>
+        <p class="pt-[3%] md:pt-0 2xl:w-3/5">
```

Comment:
Same here , if there is any other way of styling this without using arbitrary value , you may use that way instead